fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/service/ChargeEnumerations.java

```diff
@@ -191,4 +192,36 @@ public static EnumOptionData chargePaymentMode(final ChargePaymentMode type) {
         return optionData;
     }
 
+    public static EnumOptionData feeFrequencyType(final int id) {
+        return feeFrequencyType(PeriodFrequencyType.fromInt(id));
+    }
+
+    public static EnumOptionData feeFrequencyType(final PeriodFrequencyType frequencyType) {
+        EnumOptionData optionData = null;
+        switch (frequencyType) {
+            case DAYS -> {
+                optionData = new EnumOptionData(PeriodFrequencyType.DAYS.getValue().longValue(), PeriodFrequencyType.DAYS.getCode(),
+                        "Daily");
+            }
```

Comment:
Dummy question but don't you need a break here and for the other cases?

Reply:
Not with switch with arrows:
https://docs.oracle.com/en/java/javase/13/language/switch-expressions.html